examples/LinAlg/vector-matrix-proposal.hlsl

```diff
@@ -0,0 +1,143 @@
+// RUN: %dxc -I %hlsl_headers -T lib_6_9 -enable-16bit-types %s
```

Comment:
Does this actually get executed during the build?

Reply:
This location won't run it as a test. I do think it's kind of awkward to keep it given there's a copy in the proposal itself, so I'll just delete it.